services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/api/rpc/_resource_tracker.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import FastAPI
+from models_library.api_schemas_resource_usage_tracker.service_runs import (
+    ServiceRunPage,
+)
+from models_library.products import ProductName
+from models_library.resource_tracker import ServiceResourceUsagesFilters
+from models_library.rest_ordering import OrderBy
+from models_library.users import UserID
+from models_library.wallets import WalletID
+from servicelib.rabbitmq import RPCRouter
+
+from ...core.errors import CustomResourceUsageTrackerError
+from ...modules.db.repositories.resource_tracker import ResourceTrackerRepository
+from ...services.resource_tracker_service_runs import list_service_runs
+
+router = RPCRouter()
+
+
+@router.expose(reraise_if_error_type=(CustomResourceUsageTrackerError,))
```

Comment:
could you maybe double check this with @pcrespov?
If I understand this correctly that means that all errors are transformed into RPCServerError but CustomResourceUsageTrackerError which is transmitted to the client.
As far as I can see in your code here, CustomResourceUsageTrackerError is a class that exists only in resource-usage-trackerr service right? that means the RPC client will receive the error and cannot decode it.
Normally these kind of errors I would expect to be in a common package between the server and the client.

Reply:
Yes, good catch! I have added this in my last commits so it is true I do not have a test for that may be the error propagation is not working properly, I will add a test and fix that :+1:

Reply:
Yes. I missed thisone. I wonder how could we make this safer and not wait until we have the exception to find out

Reply:
done